infrastructure/network/netadapter/server/grpcserver/protowire/rpc_unban.go

```diff
@@ -0,0 +1,38 @@
+package protowire
+
+import (
+	"github.com/kaspanet/kaspad/app/appmessage"
+	"net"
+)
+
+func (x *KaspadMessage_UnbanRequest) toAppMessage() (appmessage.Message, error) {
+	return &appmessage.UnbanRequestMessage{
+		IP: net.ParseIP(x.UnbanRequest.Ip),
```

Comment:
I'd prefer if you don't hide here the logic of how to turn strings into what kaspad understands to be IPs.
Instead, appmessage.BanRequestMessage and UnbanRequestMessages should contain strings, and the conversion logic should be in the flows.